Q-And-A/Q-And-A/ViewControllers/AnswerViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AnswerViewController.swift
+//  Q-And-A
+//
+//  Created by Angel Buenrostro on 1/15/19.
+//  Copyright © 2019 Angel Buenrostro. All rights reserved.
+//
+
+import UIKit
+
+class AnswerViewController: UIViewController {
+    
+    var questionController: QuestionController? = nil
+    
+    var question: Question? = nil
+
```

Comment:
You also need to add your `updateViews` function inside of `viewWillApear` function. Because when you pass a data a that time all views is going to be loaded to the screen. So only `didSet` isn't going to work
```suggestion
override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        updateViews()
    }
```